src/Detector/Contracts/IPlatformDetector.cs

```diff
@@ -17,11 +17,5 @@ public interface IPlatformDetector
         /// <returns>An instance of <see cref="PlatformDetectorResult"/> if detection was
         /// successful, <c>null</c> otherwise</returns>
         PlatformDetectorResult Detect(DetectorContext context);
-
-        /// <summary>
-        /// Platform that the detector is used for.
-        /// </summary>
-        /// <returns>An instance of <see cref="Detector.PlatformName"/></returns>
-        PlatformName PlatformName { get; }
```

Comment:
@qianz2 since we are no longer returning a dictionary from the getalldetectedplatforms method, i removed this